src/main/java/org/folio/rest/helper/ExchangeRateHelper.java

```diff
@@ -0,0 +1,25 @@
+package org.folio.rest.helper;
+
+import io.vertx.core.Context;
+import org.folio.rest.acq.model.finance.ExchangeRate;
+
+import javax.money.convert.MonetaryConversions;
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.util.Locale;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+public class ExchangeRateHelper extends AbstractHelper {
+  public ExchangeRateHelper(Map<String, String> okapiHeaders, Context ctx, String lang) {
+    super(okapiHeaders, ctx, lang);
+  }
+
+  public CompletableFuture<ExchangeRate> getExchangeRate(String from, String to) {
+    CompletableFuture<ExchangeRate> future = new CompletableFuture<>();
+    double exchangeRate = MonetaryConversions.getExchangeRateProvider().getExchangeRate(from, to).getFactor().doubleValue();
+    String exchangeRateString = new DecimalFormat("#.###", new DecimalFormatSymbols(Locale.US)).format(exchangeRate);
```

Comment:
I dont think we want to loose precision at this point, so no addition formatting is needed from my point of view. Moreover, I don't think it comfortable to store exchangeRate as string and use it in further calculations, because this is definitely number.

Reply:
In the original requirements exchangeRate is string https://issues.folio.org/browse/MODFIN-111. Is it possible to change requirements for exchangeRate representation?

Reply:
done